jupyterlab_hdf/__init__.py

```diff
@@ -11,34 +11,29 @@
 from .data import HdfDataHandler
 from .snippet import HdfSnippetHandler
 
-path_regex = r'(?P<path>(?:(?:/[^/]+)+|/?))'
+path_regex = r"(?P<path>(?:(?:/[^/]+)+|/?))"
+
 
 def _jupyter_server_extension_paths():
-    return [{
-        'module': 'jupyterlab_hdf'
-    }]
+    return [{"module": "jupyterlab_hdf"}]
+
 
-def _load_handlers(nb_server_app, web_app):
+def _load_handlers(notebook_dir, web_app):
     # Prepend the base_url so that it works in a jupyterhub setting
-    base_url = web_app.settings['base_url']
+    base_url = web_app.settings["base_url"] if "base_url" in web_app.settings else "/"
```

Comment:
Introduced as fallback to avoid not obvious `404` when `base_url` is not present in the app settings.